packages/app/src/cli/commands/app/dev.ts

```diff
@@ -87,15 +87,15 @@ If you're using the Ruby app template, then you need to complete the following s
       exclusive: ['tunnel'],
     }),
     'use-localhost': Flags.boolean({
-      hidden: true,
+      hidden: false,
```

Comment:
false is the default value, you can remove the hidden property entirely :)

Reply:
Removed.  Also removed it everywhere else we have hidden:false for the dev command flags